src/Item.ts

```diff
@@ -278,6 +278,7 @@ export class Item extends GameEntity {
 		}
 
 		this.__manager = state.ItemManager;
+		super.hydrate(state, serialized);
```

Comment:
Should we update the type on serialized arg on EffectableEntity.hydrate() to be 'any'?

Reply:
Not sure but I'll take a look at it before merging 👍🏼

Reply:
scope creep 😎